packaging/suse/_service

```diff
@@ -1,5 +1,5 @@
 <services>
-    <service name="tar_scm" mode="disabled">
+    <service name="tar_scm" mode="manual">
```

Comment:
What is this for? Couldn't we just use `osc service un disabledrun`? What's the difference?
Was it deprecated?

Reply:
yes, this was introduced recently I think. The practical difference is not much, but basically what was previously called `disabled` should always been `manual` because that's the intention: to run these services manually. This is now done via `osc service manualrun|mr`.